module_bindings/dripline_core/run_simple_service_pybind.hh

```diff
@@ -0,0 +1,12 @@
+#include "run_simple_service.hh"
+#include "pybind11/pybind11.h"
+
+namespace dripline_pybind {
```

Comment:
our convention is that `{}` blocks are opened on the following line, with both at the same indentation as the line that creates the block, so:
```
namespace dripline_pybind
{
    <stuff>
}
```